lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -2551,111 +2551,74 @@ def model_add_psz_ac(model,
         clg_coil = nil
       end
 
-      # wrap coils in a unitary system if cycling
+      # cycling fan
       if fan_type == 'Cycling'
         # Use a Fan:OnOff in the unitary system object
         fan = create_fan_by_name(model,
                                  'Packaged_RTU_SZ_AC_Cycling_Fan',
                                  fan_name: "#{air_loop.name} Fan")
-        case heating_type
-        when 'Water To Air Heat Pump'
-          # Cycling: Unitary System
-          unitary_system = OpenStudio::Model::AirLoopHVACUnitarySystem.new(model)
-          unitary_system.setSupplyFan(fan) unless fan.nil?
-          unitary_system.setHeatingCoil(htg_coil) unless htg_coil.nil?
-          unitary_system.setCoolingCoil(clg_coil) unless clg_coil.nil?
+      # cav on/off fan
+      elsif heating_type == 'Single Speed Heat Pump' || fan_location == 'DrawThrough' || fan_location == 'BlowThrough'
+        # Use a Fan:OnOff in the unitary system object
+        fan = create_fan_by_name(model,
+                                 'Packaged_RTU_SZ_AC_CAV_OnOff_Fan',
+                                 fan_name: "#{air_loop.name} Fan")
+        fan.setAvailabilitySchedule(hvac_op_sch)
+      # invalid fan location
+      else
+        OpenStudio.logFree(OpenStudio::Error, 'openstudio.model.Model', 'Invalid fan location')
+        return false
+      end
+
+      # wrap coils in a unitary system
+      unitary_system = OpenStudio::Model::AirLoopHVACUnitarySystem.new(model)
+      unitary_system.setSupplyFan(fan) unless fan.nil?
+      unitary_system.setHeatingCoil(htg_coil) unless htg_coil.nil?
+      unitary_system.setCoolingCoil(clg_coil) unless clg_coil.nil?
+      if fan_type == 'Cycling'
+        unitary_system.setControllingZoneorThermostatLocation(zone)
```

Comment:
This can be taken out of the if statement as it applies to any value of `fan_type`.